README.md

```diff
@@ -79,7 +79,27 @@ Run unit tests and a sample solution:
     ./unit_testing
     ./maDGiCart
 
+### Singularity build from Docker
 
+If you wish to build and run using Singularity, you can convert the Docker image (built as described above) using this workflow.
+
+Get the image id from this command:
+
+    docker images
+
+Save that image as a tar file (say it was id 9c27e219663c):
+
+    docker save 9c27e219663c -o myimage.tar
```

Comment:
earlier in the readme i have it using docker tags e.g. `-t madg-cuda` and `-t madg-gcc`, you could use those instead of the SHA id.

Reply:
should be fixed now